src/file-upload/file-list.tsx

```diff
@@ -83,6 +120,36 @@ export const FileList = ({
         onItemDelete(item);
     };
 
+    const handleDragEnd = (event: DragEndEvent) => {
+        if (onReorder) {
+            const { active, over } = event;
+
+            if (active.id !== over.id) {
```

Comment:
to check `over` is defined. found if the item is dragged outside of the parent boundary, it's set to null

Reply:
But nothing actually happens yes?

Reply:
nothing happens because a TypeError is thrown 🙃